lib/Dropdown/DropdownContent.js

```diff
@@ -113,16 +113,19 @@ class DropdownContent extends Component {
       noBorder
     } = this.props;
     const { showContent } = this.context[GATHER_UI_DROPDOWN];
-    const classNames = cx(`dropdown__content ${className}`, {
-      'is-active': showContent,
-      'dropdown__content--collapse': collapse,
-      'dropdown__content--right': right,
-      'dropdown__content--centre': centre,
-      'dropdown__content--top': top,
-      'dropdown__content--fix-right': fixRight,
-      'dropdown__content--full': full,
-      'dropdown__content--noborder': noBorder
-    });
+    const classNames = cx(
+      `dropdown__content border border-solid border-neutral-90 rounded shadow ${className}`,
+      {
+        'is-active': showContent,
+        'dropdown__content--collapse': collapse,
+        'dropdown__content--right': right,
+        'dropdown__content--centre': centre,
+        'dropdown__content--top': top,
+        'dropdown__content--fix-right': fixRight,
+        'dropdown__content--full': full,
+        'dropdown__content--noborder': noBorder
```

Comment:
💚 Just checking these options all still work and tailwind doesnt override them?

Reply:
Great catch! TY - checked these, and they're all good 👍